bids/utils.py

```diff
@@ -5,13 +5,13 @@
 
 
 def listify(obj):
-    ''' Wraps all non-list or tuple objects in a list; provides a simple way
-    to accept flexible arguments. '''
+    """ Wraps all non-list or tuple objects in a list; provides a simple way
+    to accept flexible arguments. """
```

Comment:
This file appears to have been automatically reformatted. Can you please revert?

Reply:
Sure. However, It was not automatically, it was because using single triple quotes is not PEP8-compliant ("For triple-quoted strings, always use double quote characters to be consistent with the docstring convention in PEP 257."; https://www.python.org/dev/peps/pep-0008/). I did not make a comprehensive change of these, but changed them when I was encountering them. But I can revert if you prefer.